src/Gir.Tests/InterfaceTests.cs

```diff
@@ -153,6 +153,8 @@ public interface IComponent
 		/// when this object receives focus events (in or out). If the handler is
 		/// already added it is not added again
 		///</summary>
+		[Obsolete (""(Version: 2.9.4) If you need to track when an object gains or
```

Comment:
❤️

Reply:
I wasn't sure how we wanted this to look, but I figured we could fix up the formatting later